Sources/Sentry/SentryOptions.m

```diff
@@ -352,6 +383,16 @@ - (BOOL)validateOptions:(NSDictionary<NSString *, id> *)options
         self.tracePropagationTargets = options[@"tracePropagationTargets"];
     }
 
+    // TODO: do I have to check the instance type of the items?
+    // or can I do in the signature?
```

Comment:
@brustolin is this necessary? maybe the items are not of the type `SentryHttpStatusCodeRange`

Reply:
I don't feel like we need to, we haven't done it in any other property, if customers put the wrong value here the app would break during development. 

We also plan to remove this constructor. This is bad design.

@philipphofmann What do you think about this?